.changeset/itchy-spies-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+"@suspensive/suspensive.org": patch
+---
+
+docs(suspensive.org): add infiniteQueryOptions doc
```

Comment:
Remove changeset please! suspensive.org don't have to be version bump

Reply:
Oh, sorry, my mistake.